docs/source/mda_COG.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "235dee9e-d50a-4cb4-8c25-fa0f3fc33c50",
+   "metadata": {},
+   "source": [
+    "# Diffusion coefficient of molecules using center of mass\n",
+    "\n",
+    "`Kinisi` includes the ability to calculate the mean-squared displacement and diffusion coefficient of the center of mass (or geometry) of molecules. This can be done for a single molecule or a collection of molecules. It is important to note that inclusion of rotational motion in the calcuation of diffusion coeffiencents can lead to erronious results. This rotation can be elminated from the calculation by taking the center of mass for each molecule.\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "Warning\n",
+    "\n",
+    "The warnings that are being ignored are related to the parsing of the files by `MDAnalysis` and lead to unnecessary print out to the screen that we want to avoid in the web documentation.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "cc9626ec-5a7d-4be9-a23c-6721f6f4addc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\jd15489\\AppData\\Local\\miniconda3\\envs\\kinisi_dev\\Lib\\site-packages\\tqdm\\auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n",
+      "C:\\Users\\jd15489\\AppData\\Local\\miniconda3\\envs\\kinisi_dev\\Lib\\site-packages\\MDAnalysis\\topology\\TPRParser.py:161: DeprecationWarning: 'xdrlib' is deprecated and slated for removal in Python 3.13\n",
+      "  import xdrlib\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import MDAnalysis as mda\n",
+    "import matplotlib.pyplot as plt\n",
+    "from kinisi.analyze import DiffusionAnalyzer\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\", category=UserWarning)"
```

Comment:
Does an error get raised if you don't have this?

Reply:
This suppresses warnings about missing atom mass data from MDAnalysis, although these warnings are very small: 
"UserWarning: Failed to guess the mass for the following atom types: SI warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))"